Source/ZoomNet/Resources/Reports.cs

```diff
@@ -54,5 +54,45 @@ public Task<PaginatedResponseWithToken<ReportMeetingParticipant>> GetMeetingPart
 				.WithCancellationToken(cancellationToken)
 				.AsPaginatedResponseWithToken<ReportMeetingParticipant>("participants");
 		}
+
+		/// <summary>
+		/// Get a list past meetings and webinars for a specified time period. The time range for the report is limited to a month and the month must fall within the past six months.
+		/// </summary>
+		/// <param name="userId">The user ID or email address of the user.</param>
+		/// <param name="from">Start date.</param>
+		/// <param name="to">End date.</param>
+		/// <param name="type">The meeting type to query for.</param>
+		/// <param name="pageSize">The number of records returned within a single API call.</param>
+		/// <param name="pageToken">
+		/// The next page token is used to paginate through large result sets.
+		/// A next page token will be returned whenever the set of available results exceeds the current page size.
+		/// The expiration period for this token is 15 minutes.
+		/// </param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// An array of <see cref="PastMeeting">meetings.</see>.
+		/// </returns>
+		public Task<PaginatedResponseWithToken<PastMeeting>> GetMeetingsAsync(string userId, DateTime from, DateTime to, ReportMeetingType type, int pageSize = 30, string pageToken = null, CancellationToken cancellationToken = default)
+		{
+			if (to - from > TimeSpan.FromDays(30))
+			{
+				throw new ArgumentException(nameof(to), "The date range should not exceed one month.");
```

Comment:
If we end up keeping the validation and throwing our own exception, I think `ArgumentOutOfRangeException` would be even more to the point.